### DISCUSSION TITLE
Using of githubs "Projects" Feature to priorize Issues/PRs

### DISCUSSION THREAD

**Discussion by Flyingmana** [OpenMage/magento-lts#1757] [General]
As part of the discussion of having to many open Issues and PRs, there were arguments of it being hard to find something to actually work on, as many of them are in one way or another are waiting for Feedback, or belong to very specific areas not everybody is able to approach.

Now we have already a few Projects on Repository level ( https://github.com/OpenMage/magento-lts/projects ) and also some on Organization level ( https://github.com/orgs/OpenMage/projects ) (I dont really know, why some of them are duplicated there... Probably as initially github only supported one of them)

For example we have one Project related to improving our CI Pipeline: https://github.com/OpenMage/magento-lts/projects/1
I added another Project now to keep track of Tasks to make the live of contributors/Reviewers and so on easier: https://github.com/OpenMage/magento-lts/projects/9

My Question is, would this make it easier for you to find things to work on?
Do you prefer to work on specific areas,  would it makes sense and be of help to add them as Projects?
Do you any other Area where it makes sense to organize them in Projects?

**Comment by fballiano** [comment]
I think it's a good idea, although also a single project named "issues we can work on" could almost be sufficient :-D

**Reply by Flyingmana** [reply]
I added a new Project and filled with some Issues and PRs I found which should have everything to work on them.
https://github.com/OpenMage/magento-lts/projects/10
I also added a column to move tickets to, if they are missing anything to continue, so I can check from time to time, what is missing there.
